### DISCUSSION TITLE
Add support to use manual DAP configuration

### DISCUSSION THREAD

**Discussion by HeavyPunk** [fredrikaverpil/neotest-golang#217] [General]
Hi!
Thank you for awesome plugin!

I reported a motivation for this changes in PR #216 but will describe a main line at here:
At this moment test debugging only possible with [neovim-dap-go](https://github.com/leoluz/nvim-dap-go) but I already had a DAP configuration for golang and no need to install neovim-dap-go.
In PR split a DAP strategy for 2 possible ways: using neovim-dap-go (a default behavior) or using manual DAP configuration from config

If you have any question, you are always welcome!
Thank you!

**Comment by fredrikaverpil** [comment]
Closing in favour for PR: https://github.com/fredrikaverpil/neotest-golang/pull/216
